index.d.ts

```diff
@@ -0,0 +1,48 @@
+import { type FastifyJWTOptions, type VerifyPayloadType } from '@fastify/jwt'
+import { type FastifyPluginCallback, type FastifyReply, type FastifyRequest } from 'fastify'
+import { type GetJwksOptions } from 'get-jwks'
+import { type URL, type UrlObject } from 'url'
+
+export interface JWTOptions extends FastifyJWTOptions {
+  namespace?: string
+  jwks?: boolean | GetJwksOptions
+}
+
+export interface WebhookOptions {
+  url: string | URL | UrlObject
+}
+
+export type CreateSession = (request?: FastifyRequest, reply?: FastifyReply) => Promise<void>
```

Comment:
having the `reply` here matches the docs, but I can't find anywhere where it's actually used, only the `request` is used. I think we should change this to only have the request, and update the doc.

Reply:
most APIs in Fastify requires both, I tend to have them both to avoid issues in the future and breaking changes